app/helpers/layout_helper.rb

```diff
@@ -349,4 +349,8 @@ def fullscreen_button(element = "$(this).prev()")
   def authorized_associations(associations)
     associations.included_modules.include?(Authorizable) ? associations.authorized : associations
   end
+
+  def table_css_classes(classes = nil)
+    classes ? "table table-bordered table-striped table-condensed " + classes : "table table-bordered table-striped table-condensed"
```

Comment:
I'd suggest having the extra space, or doing:

    classes = "table table-bordered table-striped table-condensed" + (classes ? ' ' + classes : '')